app/src/main/java/com/kickstarter/ui/adapters/projectcampaign/ViewElementAdapter.kt

```diff
@@ -169,7 +170,7 @@ class ViewElementAdapter(
                         viewGroup,
                         false
                     ),
-                    this.lifecycle
+                    lifecycleObservable
```

Comment:
Pass a BehaviorSubject lifecycleObservable to the AudioViewElementViewHolder so that it can subscribe to lifecycle changes from the ProjectCampaignFragment.